tests/conversion/management/commands/result_harvester_test.py

```diff
@@ -70,3 +70,24 @@ def test_add_meta_data_to_job_with_out_of_bounds_exception():
         )
         logger_mock.exception.assert_called_once_with("pbf estimation failed")
         assert conversion_job.estimated_pbf_size is None
+
+
+def multiple_queue_test_parameters():
+    queue = Mock()
+    none_queue = Mock()
+    none_queue.fetch_job.return_value = None
```

Comment:
How about more informative names: e.g. `queue_with_all_jobs` and `queue_with_no_jobs`?